doc/classes/Plane.xml

```diff
@@ -151,11 +151,11 @@
 	</methods>
 	<members>
 		<member name="d" type="float" setter="" getter="" default="0.0">
-			The distance from the origin to the plane, in the direction of [member normal]. This value is typically non-negative.
+			The distance from the origin to the plane, in the direction of [member normal]. This value can be negative.
```

Comment:
Because it's said to be a distance. I mean the *"in the direction of [member normal]"* part already suggests it's a signed value so the second sentence is added just to be super clear. Can be removed of course.

The previous note I find misleading, both positive/negative values make the same sense and I'd say are equally probable. The same plane but with opposite normal will have such `d` value negated. And in which direction the normal should point is a case by case thing.

Reply:
I think it's fine as is. Distance is normally something you think of as only being positive, so a short sentence clarifying this seems perfect to me.